proto/dydxprotocol/indexer/events/events.proto

```diff
@@ -107,7 +107,18 @@ message MarketCreateEventV1 {
 
 // MarketModifyEvent message contains all the information about a market update
 // on the V4 chain
-message MarketModifyEventV1 { MarketBaseEventV1 base = 1; }
+message MarketModifyEventV1 {
+  MarketBaseEventV1 base = 1;
+
+  // Static value. The exponent of the price.
+  // For example if Exponent == -5 then a `exponent_price` of 1,000,000,000
+  // represents $10,000. Therefore 10 ^ Exponent represents the smallest
+  // price step (in dollars) that can be recorded.
+  //
+  // Updates to this value are allowed, but are unsafe - they can cause
+  // the network to have innacurate prices for up to ~1 minute.
+  sint32 exponent = 2;
```

Comment:
What's the reasoning behind adding `exponent` as a field in `MarketModifyEventV1` as opposed to `MarketBaseEventV1`?

Are we using the proto definition permanently, or just for this upgrade?

Reply:
Can we note that this is just for upgrade testing as well?

Reply:
I was trying to minimize code change footprints - this way, I only needed to update emitted MarketModifyEvents in the protocol. I agree that putting it in BaseEvent is better, but my understanding was that this would be not permanently included in the repository.